### ISSUE TITLE
Exposure of HTML elements mapped to generic

### ISSUE THREAD

**Issue by jcsteh** [w3c/html-aam#489]
Several HTML elements are mapped to ARIA role generic; e.g. `<b>`, `<i>`, `<samp>`, `<small>`, `<span>`, `<u>`. However, most (all?) browsers don't expose them in the a11y tree unless they have other properties (tabindex, etc.) that force them to be exposed. Currently, [all three browsers fail for all of these3 tags](https://wpt.fyi/results/html-aam/roles.html?label=master&label=experimental&aligned&q=firefox%3A%21pass%20chrome%3A%21pass%20safari%3A%21pass%20path%3A%2Faccessibility%20or%20path%3A%2Faccname%20or%20path%3A%2Fhtml-aam%20or%20path%3A%2Fwai-aria%20or%20path%3A%2Fsvg-aam%20or%20path%3A%2Fgraphics-aria%20or%20path%3A%2Fgraphics-aam%20or%20path%3A%2Fdpub-aam%20or%20path%3A%2Fhtml%2Fdom%2Faria-element-reflection.html).

This leads me to question whether the spec is realistic here. It's reasonable to map these to generic when they *are* included in the a11y tree. However, browsers exclude them (if inclusion isn't forced) to keep the tree from becoming needlessly deep and to improve performance.

I don't see anything in the spec that currently makes inclusion optional. [Inclusion is optional for elements which are "not mapped"](https://w3c.github.io/html-aam/#html-element-role-mappings), but no such affordance is given for generic.

**Comment by scottaohara** [comment]
IMO, I see this as a clarification for the `generic` definition in ARIA, as I don't think anyone had any intention of browsers not ignoring generics unless they had attributes important to accessibility.

Certain attributes - such as the mentioned `tabindex` - when applied to elements is being considered a signal for modifying an element's role beyond even `generic`, re: the Minimum Role proposal https://github.com/w3c/html-aam/pull/454

If others disagree with my mention that ARIA's definition of generics could be updated to indicate they can be ignored by browsers so long as they don't have important attributes for accessibility, then the proposed minimum role section seems a very good place to mention that in this spec.   Though, I would imagine this might need to be re-stated in other extension and AAM specs where they'd also likely have elements that map to generic... hence why i think that detail is worth a more global clarification than just here in HTML AAM.

**Comment by cookiecrook** [comment]
Related WPT issue at https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/48

**Comment by cookiecrook** [comment]
FYI, I'll have a WPT PR soon allowing any of "generic", "none", or "" (empty string) on all generic role tests in HTML-AAM.
